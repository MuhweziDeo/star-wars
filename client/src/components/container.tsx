import React from 'react';
import {styled} from 'baseui';

export const Container = styled("div",  ({}) => {
    return {
        height: "100vh"
    }
})