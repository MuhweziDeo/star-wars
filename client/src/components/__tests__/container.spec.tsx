import React from 'react'
import TestRenderer from 'react-test-renderer';
import { Container } from '../container';

describe('Container', () => {
    it("should render without crushing", () => {
        const container = TestRenderer.create(<Container>
            <p>hello</p>
        </Container>)
        expect(container.toJSON()).toMatchSnapshot();
        const paragraph = container.root.findByType("p");
        expect(paragraph).toBeDefined();
        expect(paragraph.props.children).toContain("hello");
    })
})