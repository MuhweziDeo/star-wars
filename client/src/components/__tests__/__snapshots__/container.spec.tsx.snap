// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render without crushing 1`] = `
<div
  className=""
>
  <p>
    hello
  </p>
</div>
`;
